ko/admin/backup.inc

```diff
@@ -175,7 +175,7 @@ backupdb
 
 압축 백업(compress backup)은 데이터베이스를 압축하여 백업을 수행하기 때문에 백업 볼륨의 크기가 줄어들어 디스크 I/O 비용을 감소시킬 수 있고, 디스크 공간을 절약할 수 있다.
 
-압축 백업을 하려면 **cubrid backupdb -z** | **--compress** 명령어를 사용한다.
+큐브리드 11.1버전부터는 압축 백업 모드가 기본값으로 설정되어 있다.
```

Comment:
The 11.1 version is not an official release version. @mhoh3963 Please confirm this.

Reply:
@hornetmj 
you are right.
@eido5 
please change 11.1 to 11.2.

Reply:
I will apply it. Thank you.